docs/static/openapi/edge-api.yaml

```diff
@@ -0,0 +1,225 @@
+      summary: Bulk insert identities (overwrite)
+      description: |+
+        Bulk uploading identities and traits. If there is an existing identity, using POST will overwrite the existing identity with the data from the request.
+
+        It has a limit of 100 identities per request.
+      requestBody:
+        content:
+          application/json:
+            schema:
+              type: object
+              example: {}
+              $ref: '#/components/schemas/BulkIdentitiesData'
+      responses:
+        '202':
+          description: Accepted
+        '400':
+          description: Bad Request
+          content:
+            text/plain:
+              schema:
+                type: string
+                description: Number of identities must not be more than 100
+        '403':
+          description: Forbidden
+          content:
+            text/plain:
+              schema:
+                type: string
+                description: Organisation is disabled from serving flags.
+        '404':
+          description: Not Found
+          content:
+            text/plain:
+              schema:
+                type: string
+                description: Environment not found
+    put:
+      tags: ['Edge API']
+      summary: Bulk insert identities (update)
+      description: |+
+        Bulk uploading identities and traits. If there is an existing identity, using PUT will update the existing identity with the data from the request.
+
+        It has a limit of 100 identities per request.
+      requestBody:
+        content:
+          application/json:
+            schema:
+              type: object
+              example: {}
+              $ref: '#/components/schemas/BulkIdentitiesData'
+      responses:
+        '202':
+          description: Accepted
+        '400':
+          description: Bad Request
+          content:
+            text/plain:
+              schema:
+                type: string
+                description: Number of identities must not be more than 100
+        '403':
+          description: Forbidden
+          content:
+            text/plain:
+              schema:
+                type: string
+                description: Organisation is disabled from serving flags.
+        '404':
+          description: Not Found
+          content:
+            text/plain:
+              schema:
+                type: string
+                description: Environment not found
+components:
+  securitySchemes:
+    EnvironmentKeyAuth:
+      type: apiKey
+      in: header
+      name: X-Environment-Key
+  schemas:
+    Flags:
+      type: object
+      properties:
+        flags:
+          type: array
+          $ref: '#/components/schemas/FeatureState'
+    FeatureState:
+      type: object
+      properties:
+        enabled:
+          type: boolean
+        feature_state_value:
+          type: Can be any of the following - string, integer, boolean, float
+          example: 'a-remote-config-value'
```

Comment:
Strictly, this should be a [`oneOf`](https://swagger.io/docs/specification/data-models/oneof-anyof-allof-not/).

Reply:
Fixed.